pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-system]
+requires = ["setuptools>=61.2", "setuptools_scm[toml]>=3.4.3"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "requests-file"
+version = "1.5.1"
+authors = [{name = "David Shea", email = "reallylongword@gmail.com"}]
+license = {text = "Apache 2.0"}
+description = "File transport adapter for Requests"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Environment :: Plugins",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 2.7",
```

Comment:
I guess 2.7 gets the boot now, and will need to be removed from the tests as well.